ArticleShapeExtractor/modules/ExportInDesignArticlesToPlaService.inc.jsx

```diff
@@ -244,6 +252,25 @@ function ExportInDesignArticlesToPlaService(
         return articleShapeJson;
     }
 
+    /**
+     * Tells whether all given page items are placed on the same spread. If this is not the case,
+     * the items can not be selected nor grouped which is required by _exportArticlePageItems().
+     * @param {Array} pageItems 
+     * @returns 
+     */
+    this._arePageItemsOnSameSpread = function(pageItems) {
+        if (pageItems.length === 0) {
+            return true;
+        }
+        var firstItemSpread = pageItems[0].parent;
+        for (var i = 1; i < pageItems.length; i++) {
+            if (pageItems[i].parent !== firstItemSpread) {
+                return false; // Different spread found
+            }
+        }
```

Comment:
Too bad `_2.jsx` doesn't add an `Array.constructor.every`

Reply:
No worries here; Soon this will all be wiped off once we have moved to UXP.